resume.md

```diff
@@ -6,7 +6,7 @@ _Charting the knowledge of the Internet, just like Galileo charted the stars._
 
 ### GitHub Trainer
 
-Teach all things *Git*, give away all the stickers, ensure world peace.
+Teach all things _Git_, give away all the stickers, ensure world peace.
 
 <!--
```

Comment:
Doh! I even had a look at this very same text before and didn't see that the comments don't end... ever. Still I am suspicious, since I spent a good time looking at these lines without finding anything special in them